src/main/java/com/mcmoddev/lib/integration/plugins/tinkers/TinkersMaterial.java

```diff
@@ -33,6 +37,8 @@
 import slimeknights.tconstruct.library.materials.HeadMaterialStats;
 import slimeknights.tconstruct.library.materials.Material;
 
+import static java.lang.Float.*;

```

Comment:
Why ? This is pointless - we don't sidestep the boxing/unboxing like this for clarity reasons.

Reply:
Changes have been pushed and should hopefully resolve the requested